lib/lolcat/cat.rb

```diff
@@ -51,8 +51,8 @@ def self.cat!
       opt :invert, "Invert fg and bg", :short => 'i', :default => false
       opt :truecolor, "24-bit (truecolor)", :short => 't', :default => false
       opt :force, "Force color even when stdout is not a tty", :short => 'f', :default => false
-      opt :version,  "Print version and exit", :short => 'v'
```

Comment:
4879 1700 4723 1999

Reply:
If you can take the necessary initiate that will be highly appreciated.